graphql/resolvers.go

```diff
@@ -2293,14 +2285,17 @@ func (r *mutationResolver) OverrideTaskDependencies(ctx context.Context, taskID
 	currentUser := MustHaveUser(ctx)
 	t, err := task.FindOneIdAndExecutionWithDisplayStatus(taskID, nil)
 	if err != nil {
-		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("error finding task %s: %s", taskID, err.Error()))
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error finding task %s: %s", taskID, err.Error()))
 	}
 	if t == nil {
 		return nil, ResourceNotFound.Send(ctx, fmt.Sprintf("cannot find task with id %s", taskID))
 	}
 	if err = t.SetOverrideDependencies(currentUser.Username()); err != nil {
 		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error overriding dependencies for task %s: %s", taskID, err.Error()))
 	}
+	// Clear DisplayStatus so GetDisplayStatus recalculates it.
+	t.DisplayStatus = ""
+	t.DisplayStatus = t.GetDisplayStatus()
```

Comment:
Instead of having to clear the display status, can we just use FindOneIdAndExecution at 2286 instead of "with display status"?  Doesn't seem like we're using it.

Reply:
Nice catch. Done.